soc/arm/nuvoton_npcx/common/reg/reg_def.h

```diff
@@ -1173,4 +1173,16 @@ struct itim64_reg {
 #define NPCX_ITCTSXX_CKSEL               4
 #define NPCX_ITCTSXX_ITEN                7
 
+/* Debug Interface registers */
+struct dbg_reg {
+	volatile uint8_t DBGCTRL;
```

Comment:
Could you add comments to indicate the full name and offset of the module register like t[his](https://github.com/zephyrproject-rtos/zephyr/blob/master/soc/arm/nuvoton_npcx/common/reg/reg_def.h#L419)?

Reply:
done.